randovania/games/planets_zebeth/pickup_database/pickup-database.json

```diff
@@ -0,0 +1,298 @@
+            "broad_category": "movement",
+            "model_name": "spr_ITEM_Jump_Boots",
+            "offworld_models": {
+                "prime1": "Space Jump Boots",
+                "prime2": "SpaceJumpBoots",
+                "dread": "powerup_doublejump",
+                "am2r": "sItemHijump"
+            },
+            "progression": [
+                "Hi-Jump"
+            ],
+            "default_shuffled_count": 1,
+            "default_starting_count": 0,
+            "preferred_location_category": "major",
+            "original_location": 16
+        },
+        "Ice Beam": {
+            "pickup_category": "beam",
+            "broad_category": "beam_related",
+            "model_name": "spr_ITEM_Ice_Beam",
+            "offworld_models": {
+                "prime1": "Ice Beam",
+                "dread": "powerup_icemissile",
+                "am2r": "sItemIceBeam"
+            },
+            "progression": [
+                "Ice Beam"
+            ],
+            "default_shuffled_count": 2,
+            "default_starting_count": 0,
+            "preferred_location_category": "major"
+        },
+        "Wave Beam": {
+            "pickup_category": "beam",
+            "broad_category": "beam_related",
+            "model_name": "spr_ITEM_Wave_Beam",
+            "offworld_models": {
+                "prime1": "Wave Beam",
+                "dread": "powerup_wavebeam",
+                "am2r": "sItemWaveBeam"
+            },
+            "progression": [
+                "Wave Beam"
+            ],
+            "default_shuffled_count": 1,
+            "default_starting_count": 0,
+            "preferred_location_category": "major",
+            "original_location": 24
+        },
+        "Long Beam": {
+            "pickup_category": "beam",
+            "broad_category": "beam_related",
+            "model_name": "spr_ITEM_Long_Beam",
+            "offworld_models": {},
+            "progression": [
+                "Long Beam"
+            ],
+            "default_shuffled_count": 1,
+            "default_starting_count": 0,
+            "preferred_location_category": "major",
+            "original_location": 2
+        },
+        "Morph Ball": {
+            "pickup_category": "morph_ball",
+            "broad_category": "morph_ball_related",
+            "model_name": "spr_ITEM_Morph",
+            "offworld_models": {
+                "prime1": "Morph Ball",
+                "dread": "powerup_morphball",
+                "am2r": "sItemMorphBall"
+            },
+            "progression": [
+                "Morph Ball"
+            ],
+            "default_shuffled_count": 0,
+            "default_starting_count": 1,
+            "preferred_location_category": "major",
+            "original_location": 0
+        },
+        "Missile Launcher": {
+            "pickup_category": "missile",
+            "broad_category": "missile_related",
+            "model_name": "spr_ITEM_Missile",
```

Comment:
missile launcher and missile ammo having the same model sounds like it can very much lead to confusion

Reply:
Will be handled when multiworld update is worked on